pymc3/tests/test_math.py

```diff
@@ -74,6 +74,23 @@ def test_cartesian():
     np.testing.assert_array_almost_equal(manual_cartesian, auto_cart)
 
 
+def test_cartesian_2d():
+    np.random.seed(1)
+    a = [[1, 2], [3, 4]]
+    b = [5, 6]
+    c = [0]
+    manual_cartesian = np.array(
+        [
+            [1, 2, 5, 0],
+            [1, 2, 6, 0],
+            [3, 4, 5, 0],
+            [3, 4, 6, 0],
+        ]
+    )
+    auto_cart = cartesian(a, b, c)
+    np.testing.assert_array_almost_equal(manual_cartesian, auto_cart)
+
```

Comment:
I would think that `assert_array_equal` is sufficient here (no almost).  I noticed the other `test_cartesian` does this so they should both be fixed

Reply:
Good point - those assertions have been changed.